docs/user/advertising/ad-blocking.rst

```diff
@@ -52,12 +52,16 @@ or simply the **Read the Docs Ads list**.
 Going ad-free
 -------------
 
-Users can go completely ad-free when logged in
-by becoming a `Gold member <https://readthedocs.org/accounts/gold/>`_
-or a `Supporter <https://readthedocs.org/sustainability/#donations>`_.
-Gold members may also completely remove advertising for all visitors to their projects.
+`Gold members <https://readthedocs.org/accounts/gold/>`_
+may completely remove advertising for all visitors to their projects.
 Thank you for supporting Read the Docs.
 
+Previously, being either a Gold member or
+`Supporter <https://readthedocs.org/sustainability/#donations>`_
+meant that a user got an ad-free experience while logged-in.
+While we didn't remove this capability, the cross-site cookies needed
+to make that work are no longer well supported by major browsers outside of Chrome.
```

Comment:
Good idea to make it a note :+1: 

```suggestion
.. note::

   Previously, being either a Gold member or
   `Supporter <https://readthedocs.org/sustainability/#donations>`_
   provided the user with an ad-free reading experience across all projects on Read the Docs while logged-in.
   However, the cross-site cookies needed to make that work are no longer supported by major browsers outside of Chrome,
   and this feature will soon disappear entirely.
```

Reply:
I had it as a note but then switched it, but I don't feel strongly. Happy to merge this suggestion.